com.vrcfury.vrcfury/Editor/VF/Feature/FullControllerBuilder.cs

```diff
@@ -253,6 +255,27 @@ public class FullControllerBuilder : FeatureBuilder<FullController> {
             // (we do this after rewriting paths to ensure animator bindings all hit "")
             from.RewriteParameters(RewriteParamName);
 
+            var smoothedParams = model.smoothedPrms.ToDictionary(
+                x => x.name == "*" ? x.name : RewriteParamName(x.name), 

```

Comment:
I don't think we should support * as an option. It's almost certainly going to break things, and (admittedly), smoothing is probably not the most performant thing in the world, so we shouldn't just encourage its adoption for absolutely everything everywhere.